### DISCUSSION TITLE
when i  use getsession  api protected route

### DISCUSSION THREAD

**Discussion by BADHON512** [nextauthjs/next-auth#7239] [Help]
### Environment

import ConnectDb from "@/backend/config/dbConnect"
import { getAllAddress, newAddress } from "@/backend/controllers/newAddressController"
import { isAuthenticated } from "@/backend/middlewares/auth"
import onError from "@/backend/middlewares/error"
import nc from "next-connect"

const handel=nc({onError})

ConnectDb()
handel.use(isAuthenticated).get(getAllAddress)
handel.use(isAuthenticated).post(newAddress)

export default handel

### Reproduction URL

[next-auth][error][CLIENT_FETCH_ERROR] https://next-auth.js.org/errors#client_fetch_error undefined {   error: {},   url: 'http://localhost:3000/api/auth/session',   message: undefined }

### Describe the issue

import { getSession } from "next-auth/react"
import ErrorHandle from "../Utils/errorHandler"

const isAuthenticated=async(req,res,next)=>{
    const session= await getSession({req})
    if(!session){
        return next(new ErrorHandle("Login first to access this route", 512))
    }
     req.users=session.user

    next()
}

export {isAuthenticated}

### How to reproduce

[next-auth][error][CLIENT_FETCH_ERROR]
https://next-auth.js.org/errors#client_fetch_error undefined {
  error: {},
  url: 'http://localhost:3000/api/auth/session',
  message: undefined
}

### Expected behavior

[next-auth][error][CLIENT_FETCH_ERROR]
https://next-auth.js.org/errors#client_fetch_error undefined {
  error: {},
  url: 'http://localhost:3000/api/auth/session',
  message: undefined
}

**Comment by Crazypersonalph** [comment]
Good morning,
There is a lot wrong with this issue report
Please follow the issue guide and update this issue
We need a bit more context
Thanks
